backend/src/test/java/com/barmjz/productivityapp/validators/EmailValidatorTest.java

```diff
@@ -0,0 +1,64 @@
+package com.barmjz.productivityapp.validators;
+
+import com.barmjz.productivityapp.user.registration.validators.EmailValidator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EmailValidatorTest {
+
+    private EmailValidator emailValidator;
+
+    @BeforeEach
```

Comment:
- I think, you don't have to reinitialize emailValidator before each test case.
- You can annotate it the emailValidator with @autowired annotation.

Reply:
Wouldn't work
![image](https://user-images.githubusercontent.com/73697437/205454273-e8ed9c7f-ec2b-45ff-bbee-4e1f81400715.png)